From a08fa20a731b5a1ffa197987373b9fb65b9c86cb Mon Sep 17 00:00:00 2001
From: Joel Lopes Da Silva <joel@lopes-da-silva.com>
Date: Tue, 22 Jan 2019 12:33:05 -0800
Subject: [PATCH] Revert "vimpager: remove options"

This commit enables unconditionally the step that builds man pages when
building from source. The issue here is that building those man pages is
done using pandoc, which depends on ghc, the Haskell compiler; and
building the Haskell compiler can easily take several hours.

For this reason, I would just rather revert that commit, thus making
building the man pages optional again, which will greatly improve build
times for any user who wants to build vimpager from source without it
taking several hours.

This reverts commit 039a582e716659936a23937ffb8e4cdf73815ac4.
---
 Formula/vimpager.rb | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Formula/vimpager.rb b/Formula/vimpager.rb
index 03fccffdd7b5..c678e1de666f 100644
--- a/Formula/vimpager.rb
+++ b/Formula/vimpager.rb
@@ -15,11 +15,12 @@ class Vimpager < Formula
     sha256 "4e751d2207b8925e1c229edb88a7f635d41aa611a576a1b7a9bf0b9b701df341" => :mavericks
   end
 
-  depends_on "pandoc" => :build
+  option "with-pandoc", "Use pandoc to build and install man pages"
+  depends_on "pandoc" => [:build, :optional]
 
   def install
     system "make", "install", "PREFIX=#{prefix}"
-    system "make", "docs"
+    system "make", "docs" if build.with? "pandoc"
   end
 
   def caveats; <<~EOS
-- 
2.20.1

